contents/go/Go.md

```diff
@@ -0,0 +1,261 @@
+
+Go is an [open source project](https://github.com/golang/go). Its source code may be useful reading for those interested in learning good practices, or simply to find out more about how the language was implemented.
+
+## Getting started
+
+Go provides its own [installation guide](https://golang.org/doc/install) and an interactive [ tour of Go](https://tour.golang.org/). These are useful and highly comprehensive resources for programmers looking to learn the syntax and style of Go. For those who prefer to read existing code examples, [Go by Example](https://gobyexample.com/) is a collection of code samples covering a wide variety of features in Go, and includes line-by-line explanations of the code. For those looking for a quick crash course on Go syntax, the Learn X In Y Minutes [Go cheatsheet](https://learnxinyminutes.com/docs/go/) may also be a good starting point.
+
+If you're unconvinced about Go, you can use [the Go playground](https://play.golang.org/) to write, build and execute code without installing Go on your machine.
+
+## Using Go
+As it builds on the foundations set by many popular and widely-used languages such as C, C++, Java and Python, much of Go's syntax draws from existing implementations and will be familiar to programmers looking to learn an additional language. However, Go also diverges explicitly from these other languages; listed below are some aspects of Go that may be unfamiliar to learners.
+
+### Declaring Variables
+One immediately obvious difference between Go and other languages is that Go declares its variables in the format `var variableName (variableType)`, with the variable's type declared to the right of the variable name, as follows:
+
+```go
+var name string
+name = "John Smith"
+```
+This differs from many other languages, which put the variable type to the left of the variable name.
+
+In addition, the verbosity of a declaration statement in Go can vary. You do not need to declare the `variableType` of a new variable if you use an initializer, as the type of the variable will be inferred from its initialization. For instance, in the following example, the `string` type is optional:
+
+```go
+var name string = "John Smith"
+
+// alternatively
+var name = "John Smith"
+```
+
+When an initializer is not used, declaring a variable allocates storage for the variable and initializes its value to that type's zero value. The zero value is `0` for numeric types, `false` for booleans, `""` for strings, `nil` for pointers, functions, interfaces, slices, channels, and maps.
```

Comment:
Can do without this sentence? Feels like its something the reader may not be interested at this stage.